assets/js/admin/payment-method-restrictions.js

```diff
@@ -0,0 +1,74 @@
+jQuery( function( $ ) {
+
+	/**
+	 * If the WC core validation passes (errors removed), check our own validation.
+	 */
+	$( document.body ).on( 'wc_remove_error_tip', function( e, element, removed_error_type ) {
+		var type_error = 'i18n_zero_subscription_error';
+
+		// Exit early if it's this error that has been removed.
+		if ( removed_error_type === type_error ) {
+			return;
+		}
+
+		// We're only interested in product price input.
+		if ( ! $( element ).is( '#_subscription_price' ) && ! $( element ).hasClass( 'wc_input_subscription_price' ) ) {
```

Comment:
I'm thinking we probably need to target the sale price field too?

Reply:
Yeah that's a good catch. Allowing $0 sale prices would just act as loophole if we allowed it.